### PR TITLE
Issue #20452: Add tests for the new MethodReference APIs

### LINKED ISSUE #20452: Expression Language 5.0: Add the MethodReference class to provide access to details of the method to which a MethodExpression resolves, including any annotations present on the method.
Spec issue: `https://github.com/jakartaee/expression-language/issues/154`

See "A.1. Changes between 5.0 and 4.0" here: https://github.com/jakartaee/expression-language/blob/master/spec/src/main/asciidoc/ELSpec.adoc

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #20452

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#libby #build

File: dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/test-applications/EL50MethodReferenceTest.war/src/io/openliberty/el50/fat/method/reference/servlet/EL50MethodReferenceServlet.java [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+        assertTrue("The setValueMethodReference MethodInfo was null and should not have been.", setValueMethodReference.getMethodInfo() != null);
+
+    }
+
+    /**
+     * https://jakarta.ee/specifications/expression-language/5.0/apidocs/jakarta.el/jakarta/el/methodexpression#getMethodReference(jakarta.el.ELContext)
+     *
+     * NullPointerException - If the supplied context is null
+     *
+     * @throws Excpetion
```
Comment (volosied) [inline dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/test-applications/EL50MethodReferenceTest.war/src/io/openliberty/el50/fat/method/reference/servlet/EL50MethodReferenceServlet.java]: Misspelt

**Review by volosied — APPROVED** [review]
Only comment was the spelling. 

Tests are quite thorough. Thanks for making them!

**Comment by pnicolucci** [comment]
#libby #build

**Pull request merged**